### PR TITLE
feat(plugin): RoleColorEverywhere (RCE)

### PR DESCRIPTION
**Pull request by 12944qwerty**
currently only user mentions and typing colored

todo:
 - make typing users work with i18n
 - vc users
 - slate mentions
 - statuses
 - popouts

### CODE REVIEW

**Comment by janisslsm** [comment]
RCE my beloved

File: src/plugins/roleColorEverywhere.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        }
+
+        return member.colorString
+    },
+    getUserColor({ id: userId }, channelId) {
+        const colorString = this.getColor(userId, channelId);
+        return colorString && parseInt(colorString.slice(1), 16);;
+    },
+    typingUsers(users, userIds, SEVERAL_USERS_TYPING) { // todo: work with i18n
+        const several = Object.keys(userIds).length > 3;
```
Comment (Vendicated) [inline src/plugins/roleColorEverywhere.tsx:70]: would be cool to have a setting to show more useful info like `Tom, Jerry and 7 more people are typing`

**Comment by Vendicated** [comment]
Is this ready yet?

**Comment by lewisakura** [comment]
finished in #482

**Pull request closed without merging**